### PR TITLE
Vulpkanin emotes

### PR DESCRIPTION
**Pull request by Allfd**
This is the second round of dehumanizing of Vulpkanin, I have separated this from the changes Tajarans both to keep the PRs atomic, and due to the lively discussion I am anticipating.

My goal in this PR was to follow up on the Vulp and Taj color blindness PR with more changes aimed at altering these species both cosmetically and mechanically to distance them from humans.

Special thanks to @Anticept for help with editing the sound files.

🆑Alffd
Add: Ported growl and howl sounds from Goon
Add: Added growl and howl emotes to Vulpkanin
/🆑

### CODE REVIEW

**Comment by Anticept** [comment]
Everything but the yelp is really loud and should be turned down a bit.

**Comment by Xantholne** [comment]
If there is no *awoo we have an issue, @Allfd

**Comment by KasparoVy** [comment]
Will conflict with #7105 but resolving it on my end won't be an issue at all, just documenting it for posterity

**Comment by Ty-Omaha** [comment]
Any preview of the sounds?

**Comment by KasparoVy** [comment]
@Ty-Omaha https://github.com/ParadiseSS13/Paradise/pull/7110/files and scroll down, I'm pretty sure you can listen to all of them in-browser.

**Comment by Allfd** [comment]
@Anticept amplitude of the goonsounds reduced to match that of the other resources and existing goon resources.

**Comment by Ty-Omaha** [comment]
@KasparoVy Worked with firefox, thanks.

I like them :+1:

**Comment by KasparoVy** [comment]
The reduced length is a pretty big improvement. Sounds great

**Comment by Jovaniph** [comment]
The yelp doesn't sound good. Everything else is fine.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -85,7 +95,40 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if(param)
+				message = "<B>[src]</B> growls at [param]!"
+			else
+				message = "<B>[src]</B> growls!"
+			playsound(src.loc, "growls", 80, 0)
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/emote.dm]: no src. please

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -85,7 +95,40 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if(param)
+				message = "<B>[src]</B> howls at [param]!"
+			else
+				message = "<B>[src]</B> howls!"
+			playsound(src.loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/emote.dm]: no src. please

**Comment by Allfd** [comment]
@tigercat2000 src. removed.

**Comment by Anticept** [comment]
@Jovaniph that's what foxes actually sound like. The rest of the sounds are more wolves.

**Comment by taukausanake** [comment]
@Anticept I don't know, I'd go for that Barking Fox as a vulp scream over human scream any day

**Comment by KasparoVy** [comment]
@Jovaniph Vox shrieking isn't exactly music to the ears, is it?

**Comment by TravellingMerchant** [comment]
Can you please make a youtube video or some other example that we can hear them with?

**Comment by Anticept** [comment]
@TravellingMerchant go to view source and check the .oggs at the bottom.

**Comment by TravellingMerchant** [comment]
Does not work for me. They wont play.

**Comment by taukausanake** [comment]
The sounds don't play nice in Chrome. I request video too

**Comment by Anticept** [comment]
Load up github desktop and click the button "view in github desktop"

**Comment by tigercat2000** [comment]
>using github desktop

You can just download these files temporarily by clicking on "View > View Raw", you know :stuck_out_tongue:

**Comment by Anticept** [comment]
Nope. Doesn't work for me like that either. The github web client is acting really weird in that regard.

**Comment by KasparoVy** [comment]
Watt le'fuque?

**Comment by Anticept** [comment]
Now, opening an incognito window, then right clicking and hitting "save link as" on the download button works!

**Comment by KasparoVy** [comment]
LOL. At least there's something.

**Comment by Spacemanspark** [comment]
There was a PR that ported Goon's growl/ howl noises in the past, and it was rejected because the sounds were... eh.
In addition, I really don't like that the scream noise sounds like an old lady about to get chainsawed by a bad guy in a horror movie.

**Comment by SamHPurp** [comment]
Played with this on a server, @Allfd it sounds great - however, there is a big delay from the scream audio happening after beginning to scream. I expect its too much empty space at the beginning of the audio clip?

**Comment by KasparoVy** [comment]
Easy fix, only gotta trim off the first second (or however long) where the silence is in the vulp_yelp.ogg

**Comment by KasparoVy** [comment]
Easy fix, only gotta trim off the first second (or however long) where the silence is in the vulp_yelp.ogg

**Comment by Anticept** [comment]
Alright during a test server run of some other code, it turns out the weird vulp yelp actually fits pretty better than I thought it would. Especially once you light the snowflake on fire! But no really, it sounds a lot better once the violence starts.

**Comment by Allfd** [comment]
I cleaned up the yell so that it will start immediately.

I am open to any suggestions of alternatives, I do want to point out again that this is actually what foxes sound like.

**Comment by SamHPurp** [comment]
Thanks for making the change @Allfd .

I have to say, I was against this PR I thought it was snowflakey and not necessary yada-yada-yada.

I was wrong.

I was using this on the test server, and actually it kinda fits, and I actually quite like it.

**Comment by FlattestGuitar** [comment]
I won't mind seeing this merged too much, but I really don't see people using it to a large extent. Vulpkanin are usually played as, well, civilized characters, not dogs walking on two legs. Not many people use growls, and I'm a tad afraid this could cause some weird groups of howling vulps in maint, but that's just hypothetic.

We'll see, I suppose!

**Comment by SamHPurp** [comment]
@FlattestGuitar it changes their normal *scream.

But yeah I'm still quite against the Howl.

**Comment by KasparoVy** [comment]
Contrarily, I love the howl and the rest of the additional sound emotes. :)

**Comment by taukausanake** [comment]
Same here. I like them so far, just had to download them in IE to hear. It's a shame Tajaran can't get good feline sounds without sounding bad

**Comment by ZomgPonies** [comment]
Let's make sure this has the same cooldown as ALL OUR OTHER emotes, animations and sounds. You know, so we actually follow our established standards.

**Comment by KasparoVy** [comment]
I thought the howl was on a longer cooldown for a reason or something.

**Comment by Allfd** [comment]
Yeah I am not sure if you @ZomgPonies  want the cool down shortened or you are meaning that it should be rate limited so it can't be spammed.

**Comment by ZomgPonies** [comment]
@Allfd I'd just like to see a standard set for cooldowns of this type, for both this PR and the goddamn slap one. I am tired of the favoritism when we already have a precedent set.

**Comment by KasparoVy** [comment]
Would it be acceptable to make a define for the diff cooldown type?
i.e. the current one is EMOTE_COOLDOWN (20 deciseconds) [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/emote.dm#L1)
where the long one (i.e. howls) being EMOTE_COOLDOWN_LONG (100 ds)

`define EMOTE_COOLDOWN 20 `
`define EMOTE_COOLDOWN_LONG 100`

**Comment by Allfd** [comment]
@ZomgPonies @KasparoVy 

That change is out of scope of this PR although its something we should probably do.

Its a much bigger discussion then adding a new emote and would likely be inappropriate to append given the subject of this PR and the likely wider interest in such a cooldown.

**Comment by SamHPurp** [comment]
@Allfd that one is significantly worse, in terms of content, as well as quality... it sounds like it was recorded by a tin can...

**Comment by KasparoVy** [comment]
RIP, the first one was fantastic

**Comment by KasparoVy** [comment]
RIP, the first version of the sound was fantastic. Please reconsider your last commit.

**Comment by SamHPurp** [comment]
I have to agree with @KasparoVy

**Comment by KasparoVy** [comment]
Silent treatment, rough

**Comment by KasparoVy** [comment]
Go figure.

I can't help but think there's something we're not hearing here on the PR.

**Comment by KasparoVy** [comment]
Title still describes a feature that was cut from the PR.

**Comment by Fox-McCloud** [comment]
🦊 🐶 🐺  Awoooooo

**Comment by KasparoVy** [comment]
Bless mon bless

**Comment by Vivalas** [comment]
>Let's add these horrible sounds to vulp
>Naah, they sound horrible
>Wait a bit...
>Let's add these horrible sounds to vulp! (no difference, we just waited)
>Okay! 

image

**Pull request merged**